app/models/audit/SubmitExplainDelayAudit.scala

```diff
@@ -46,8 +46,7 @@ case class SubmitExplainDelayAudit(
       case Right(success) =>
         Json.obj(fields =
           "status" -> "success",
-          "receipt" -> success.receipt,
-          "receiptDate" -> success.receiptDate
```

Comment:
These audit events would have been signed off by CIP, I assume. I think we still need to have the receiptDate (date/time of now) otherwise we may have to trigger a reassessment

Reply:
Adding back in `receiptDate` https://github.com/hmrc/emcs-tfe-explain-delay-frontend/pull/32/commits/a92da4648945a325a2b40518ab1da0ca406a20e1